lib/index.js

```diff
@@ -16,30 +16,45 @@ var Yandex = module.exports = integration('Yandex Metrica')
   .assumesPageview()
   .global('yandex_metrika_callbacks')
   .global('Ya')
+  .option('type', 0)
   .option('counterId', null)
   .option('clickmap', false)
   .option('webvisor', false)
+  .option('trackHash', false)
+  .option('trackLinks', false)
+  .option('accurateTrackBounce', false)
+  .option('params', null)
```

Comment:
We are using it basically for two things
A/B testing https://yandex.com/support/metrica/reports/visit-params.xml
and tracking Goals specific parameters https://yandex.com/support/metrica/objects/reachgoal.xml#reachgoal